package.json

```diff
@@ -28,7 +28,7 @@
     "test:ie": "lerna run test:ie --concurrency 1 --stream",
     "update:dependency": "node scripts/update-dependency.js",
     "update:core": "cd jupyterlab && node update-core.js",
-    "watch": "watch \"npm run build:packages\" ./packages/** --wait 10 --filter=scripts/watch-filter.js --ignoreDotFiles",
```

Comment:
Can watch-filter.js be removed entirely now?

Reply:
Removed now, we were also using it in `test/`.